discovery/marathon/marathon.go

```diff
@@ -98,6 +100,7 @@ func NewDiscovery(conf *config.MarathonSDConfig) (*Discovery, error) {
 		servers:         conf.Servers,
 		refreshInterval: time.Duration(conf.RefreshInterval),
 		appsClient:      fetchApps,
+		token:           os.Getenv(conf.TokenEnvName),
```

Comment:
this way i would be able to run prometheus in docker with a cli argument, hardcoded token is difficult to secure and manage, do you have a better option than that to handle it ?